tar5/cup_of_join.py

```diff
@@ -0,0 +1,24 @@
+def join(*lists, sep='-'):
+      """
+    :param lists:
+    :param sep: to be between the lists

```

Comment:
When we document a function, we want to make sure that everyone who reads the code will understand it as well as possible. Hence, we can detail a bit more about the parameter sep.